stac_fastapi/types/stac_fastapi/types/rfc3339.py

```diff
@@ -122,6 +122,9 @@ def str_to_interval(interval: Optional[str]) -> Optional[DateTimeType]:
             detail="Interval string contains more than one forward slash.",
         )
 
+    if len(values) == 1:
+        values = [values[0], values[0]]
```

Comment:
well I think this wasn't really what was need 😓 
I think https://github.com/stac-utils/stac-fastapi/pull/692 does it better